tools/test/test_depsolve.py

```diff
@@ -1497,6 +1425,75 @@ def test_get_test_case_repo_servers(test_case, repo_servers, expected):
     assert get_test_case_repo_servers(test_case, repo_servers) == expected
 
 
+def check_depsolve_api_v1_response(res, expected_pkgs, expected_repos, expected_modules, with_dnf5, with_sbom):
```

Comment:
(super nitpick) maybe `assert_depsolve_...` here to emphasize that it will assert itself instead of returning some check bool/error that needs to be asserted on the call side? But I'm probably overthinking this :)

Reply:
Thanks for the suggestion. I like the name with "assert" more. I'll make sure to unify the names.

Reply:
Done